blogs/whats_new_1.6/index.md

```diff
@@ -0,0 +1,231 @@
+
+![Subnet labels configuration](images/subnet_labels_configuration.png)
+    _<div style="text-align: center">Figure 8: Subnet labels configuration</div>_
+
+   By default, the OpenShift auto-detect mode is enabled to identify internal CIDRs as "Pods", "Services", "Machines" or "n/a" for not applicable.  While CIDRs can be internal or external, they are typically external to allow you to provide a custom name for them.  If you use a naming convention such as prefixing "Ext_" for all cluster-external traffic, then you can create a custom metric for this.  Here's a PromQL that displays a graph of external traffic in bytes over a 1-minute period.
+
+   ```
+   sum(rate(netobserv_workload_egress_bytes_total{DstSubnetLabel=~"|Ext_.*"}[1m]))
+   ```
+
+   This assumes the `workload_egress_bytes_total` metric has been included in `processor.metrics.includeList`.  See my [Network Observability 1.5 blog](https://developers.redhat.com/articles/2024/03/20/whats-new-network-observability-15) on how to do this.
+
+   In the Traffic flow table, this is what you will see if you add the Subnet Label columns (Figure 9).
+
+![Subnet labels columns](images/subnet_labels_columns.png)
+    _<div style="text-align: center">Figure 9: Subnet Label columns</div>_
+
+
+## Custom Metrics
+
+In my [Network Observability 1.5 blog](https://developers.redhat.com/articles/2024/03/20/whats-new-network-observability-15), I covered the development preview of the FlowMetrics API.  While it's possible to take any [flow data labels](https://docs.openshift.com/container-platform/4.16/observability/network_observability/json-flows-format-reference.html) and turn it into a Prometheus metric, that web page was updated to indicate which fields can be safely used as labels to avoid high cardinality.
+
+This feature is now GA, and includes more filtering options (see [reference](https://github.com/netobserv/network-observability-operator/blob/main/docs/FlowMetric.md)) and support for [dashboards in OpenShift](https://github.com/netobserv/network-observability-operator/blob/main/docs/Metrics.md#charts-openshift-only).  That link also has many more examples of using custom metrics.
+
+
+## Network Observability CLI
+
+This release has yet another significant technical preview feature called Network Observability CLI (NetObserv CLI).  It is a kubectl plug-in that effectively installs Network Observability from the command line but is text-based only.  When you exit the program, it cleans up and removes itself.  In other words, you can now run Network Observability on-demand!
+
+While it reuses much of the NetObserv code, it is separate from the Network Observability Operator.  This means even if you have NetObserv running, you can still run the NetObserv CLI independently.  For this preview, it only supports nodes running x64.
```

Comment:
```suggestion
While it reuses much of the NetObserv code, it is separate from the Network Observability Operator.  This means even if you have NetObserv running, you can still run the NetObserv CLI independently.  For this preview, it only supports nodes running `x86_64` architecture.
```

Reply:
So, in the Red Hat Style Guide, we aren't supposed to refer to this as just x64: https://redhat-documentation.github.io/supplementary-style-guide/#x86_64

Up to you to take it or no.